app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    route = command[2].split("/")
+    if command[0] != "mv":
```

Comment:
Code Efficiency: You are checking if the command is 'mv' after splitting the command. It would be more efficient to check if the command starts with 'mv ' before splitting it. This way, you avoid unnecessary operations if the command is not 'mv'.